packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -272,6 +278,55 @@ async function approveWeth() {
   await tx.wait()
 }
 
+async function approveErc20(l1ERC20Token: Contract) {
```

Comment:
I'm curious, why do we preapprove the token?

Reply:
We pre-approve the token because currently our deposit E2E tests don't test for approval flows :) 
But we don't pre-approve MAX amount because our `approveToken` test tests for that